doc/source/whatsnew/v0.24.0.txt

```diff
@@ -182,6 +182,8 @@ Other Enhancements
 - :func:`to_timedelta` now supports iso-formated timedelta strings (:issue:`21877`)
 - :class:`Series` and :class:`DataFrame` now support :class:`Iterable` in constructor (:issue:`2193`)
 - :class:`DatetimeIndex` gained :attr:`DatetimeIndex.timetz` attribute. Returns local time with timezone information. (:issue:`21358`)
+- Implement interpolating ``NaT`` values in ``datetime`` series (:issue:`11701`)
```

Comment:
The wording here is awkward.  It doesn't need to be a complete sentence per se, but take a look at the other entries for examples.